community-guidelines.md

```diff
@@ -0,0 +1,51 @@
+# Polyhack Community Guidelines
+
+**[#polyhack](https://github.com/polyhack)** is a community of
+programmers based on the IRC freenode network. It is specifically a
+polyglot group concerned to discuss programming ideas over the long-term.
+
+Its membership is predominantly from the Australian east coast but
+includes members from North America, New Zealand, and further. It was
+created in August, 2012.
```

Comment:
Does being so specific about where people come from make non East coast Australian people second class? Maybe language should sound more inviting.